mixer/tools/codegen/pkg/modelgen/model.go

```diff
@@ -338,10 +338,6 @@ func (m *Model) addTopLevelFields(fd *FileDescriptor) {
 
 	if tmplVariety, err := proto.GetExtension(fd.GetOptions(), tmpl.E_TemplateVariety); err == nil {
 		m.VarietyName = (*(tmplVariety.(*tmpl.TemplateVariety))).String()
-	} else {
```

Comment:
i don’t think this is better code, removing errors to raise coverage is counter productive

Reply:
Yes, I agree witth the sentiment. however, the code I deleted can never get executed because the caller ensures that (things would have failed long before).

Reply:
until the caller changes... and we spend time figuring out what happens

Reply:
In this case, the private function makes some assumption and the caller needs to abide to those assumption. The callee should not re-validate what the very tightly coupled caller has already done.